Data/Migrations/20211207224434_addLikedItems.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace kinder_app.Data.Migrations
+{
+    public partial class addLikedItems : Migration
```

Comment:
Im not sure, but class name should be from uppercase

Reply:
it's a migration class, and i low-key fear that I'll mess up db again. I'll try to fix it day before assignment